frontend/src/components/editor/actions/useNotebookActions.tsx

```diff
@@ -299,6 +300,15 @@ export function useNotebookActions() {
 
     {
       divider: true,
+      icon: <Home size={14} strokeWidth={1.5} />,
+      label: "Return home",
+      hidden: !location.search.includes("file"),
```

Comment:
@mscolnick can you comment whether this is an acceptable way to check whether the server is capable of serving the "home" page?

Reply:
Yea this is fine. Potentially in a future change, we should allow going back home regardless, and just use the current directly for "marimo new"